docs/PR_GUIDE.md

```diff
@@ -0,0 +1,56 @@
+# Pull Request Guide
+
+## General development rules:
+
+- All the changes of the code base are implemented through pull requests (PR) on master or other branches
+- When all the features for an upcoming release are implemented and are stable enough, a new release branch is made from master. After that, all app versions are updated in the master branch in files: ./omim/xcode/common.xcconfig and ./omim/android/gradle.properties
```

Comment:
break this line?

Reply:
I don't know... Do you suggest to break all the long lines?

Reply:
Yes, per @alexzatsepin's suggestion

Reply:
Done